DNN Platform/Library/Entities/Portals/PortalSettings.cs

```diff
@@ -179,93 +179,226 @@ private void BuildPortalSettings(int tabId, PortalInfo portal)
+        public int HostSpace { get; set; }
 
-		public string KeyWords { get; set; }
+        /// <summary>
+        /// A string containing keyworks for the built-in seach indexer and search engines
+        /// </summary>
+        public string KeyWords { get; set; }
 
-		public int LoginTabId { get; set; }
+        /// <summary>
+        /// The tab (page) ID of the login page
+        /// </summary>
+        public int LoginTabId { get; set; }
 
-		public string LogoFile { get; set; }
+        /// <summary>
+        /// The logo image file for this portal, can be used by themes and is used by the logo skinObject
+        /// </summary>
+        public string LogoFile { get; set; }
 
-		public int PageQuota { get; set; }
+        /// <summary>
+        /// If hosting fees are enabled, this would be the page quota allowed in the subscription
+        /// </summary>
+        public int PageQuota { get; set; }
 
-		public int Pages { get; set; }
+        /// <summary>
+        /// The number of pages this portal contains
+        /// </summary>
+        public int Pages { get; set; }
 
-		public int PortalId { get; set; }
+        /// <summary>
+        /// The ID of this portal (site)
+        /// </summary>
+        public int PortalId { get; set; }
 
-		public PortalAliasInfo PortalAlias { get; set; }
+        /// <summary>
+        /// The portal alias for this portal
+        /// </summary>
+        public PortalAliasInfo PortalAlias { get; set; }
 
-		public PortalAliasInfo PrimaryAlias { get; set; }
+        /// <summary>
+        /// The default portal alias for this portal
+        /// </summary>
+        public PortalAliasInfo PrimaryAlias { get; set; }
 
-		public string PortalName { get; set; }
+        /// <summary>
+        /// The name of this portal (site), used in menus, messages, logs, etc.
+        /// </summary>
+        public string PortalName { get; set; }
 
-		public int RegisteredRoleId { get; set; }
+        /// <summary>
+        /// The role ID that represents registered users
+        /// </summary>
+        public int RegisteredRoleId { get; set; }
 
-		public string RegisteredRoleName { get; set; }
+        /// <summary>
+        /// The role name that represents registered users
+        /// </summary>
+        public string RegisteredRoleName { get; set; }
 
-		public int RegisterTabId { get; set; }
+        /// <summary>
+        /// The tab (page) id of the registration page
+        /// </summary>
+        public int RegisterTabId { get; set; }
 
+        /// <summary>
+        /// The registration settings for this portal
+        /// </summary>
         public RegistrationSettings Registration { get; set; }
 
+        /// <summary>
+        /// The tab (page) id where the search results are located
+        /// </summary>
         public int SearchTabId { get; set; }
 
         [Obsolete("Deprecated in 8.0.0. Scheduled removal in v10.0.0.")]
         public int SiteLogHistory { get; set; }

```

Comment:
FYI - This leads me to believe that you aren't current with Development or I missed something on removal for 10.x

Reply:
Oh, did that get merged? There was a test failure last time...

Reply:
Yeah, that is still pending https://github.com/dnnsoftware/Dnn.Platform/pull/2869

Reply:
Well crap, I guess I should fix that

Reply:
That PR did not get merged yet...

Reply:
Sorry for the repeat, not trying to badger you, github was not showing me previous messages...